reports-scheduler/src/main/kotlin/com/amazon/opendistroforelasticsearch/reportsscheduler/action/ReportDefinitionActions.kt

```diff
@@ -74,8 +76,13 @@ internal object ReportDefinitionActions {
         UserAccessManager.validateUser(user)
         val currentReportDefinitionDetails = ReportDefinitionsIndex.getReportDefinition(request.reportDefinitionId)
         currentReportDefinitionDetails
-            ?: throw ElasticsearchStatusException("Report Definition ${request.reportDefinitionId} not found", RestStatus.NOT_FOUND)
+            ?: run {
```

Comment:
may be use if condition instead of run for simplicity and readability?

Reply:
I feel using the `run` constructor is more readable.